src/main/java/racingcar/model/Cars.java

```diff
@@ -0,0 +1,22 @@
+package racingcar.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Cars {
+
+    private List<Car> cars;
+
+    public Cars() {
+        cars = new ArrayList<>();
+    }
+
+    public void setCarNumber(int number) {
+        for (int i = 0; i < number; i++) {
+            cars.add(new Car());
+        }
+    }
+    public List<Car> getCar() {
+        return cars;
```

Comment:
Cars 밖에서 List<Car> 에 영향을 못 미치게 해주세요! 
https://velog.io/@ohzzi/collection%EC%9D%98-%EB%B3%B5%EC%82%AC-new-copyOf-unmodifiable

```suggestion
        return Collections.unmodifiableList(this.cars);
```